Util/LinkExtensions.cs

```diff
@@ -36,7 +37,8 @@ public static MvcHtmlString ResourceLink(this HtmlHelper helper, string path)
         /// <returns></returns>
         public static MvcHtmlString ResourceLink(this HtmlHelper helper, string path, string domain)
         {
-            return new MvcHtmlString(String.Format("//{2}.{0}/{1}", ConfigurationManager.AppSettings["Domain"], path, domain));
+            var port = ConfigurationManager.AppSettings["Port"];
+            return new MvcHtmlString($"//{domain}.{ConfigurationManager.AppSettings["Domain"]}:{port}/{path}");
```

Comment:
Please only add the port if it's not 80 (if the current request is https) or 443 (if the current request is via https)